src/add-text-track-data.js

```diff
@@ -40,6 +40,16 @@ const deprecateOldCue = function(cue) {
   });
 };
 
+const durationOfVideo = function(duration) {
```

Comment:
I think it would be a good idea to export this function and import it in html-media-source.js since you repeat the same logic